src/timezone.ts

```diff
@@ -44,7 +44,8 @@ export const timezone = async (message: Message) => {
       availableZones.forEach((item) =>
```

Comment:
I'll split the loop to check in chunks of three
(one passed normal-- 2 passed inline-- one passed normal-- 1 passed inline)
That would make it look neater than before(all in one horizontal row)